Tests/test_align.py

```diff
@@ -30,172 +32,968 @@
-          'T': 0.25}
-
-e_freq_table = FreqTable.FreqTable(e_freq, FreqTable.FREQ,
-                                   IUPAC.unambiguous_dna)
-
-print('relative information: %0.2f'
-      % align_info.information_content(e_freq_table=e_freq_table,
-                                       chars_to_ignore=['N']))
-
-print('Column 1: %s' % align_info.get_column(1))
-print('IC for column 1: %0.2f' % align_info.ic_vector[1])
-print('Column 7: %s' % align_info.get_column(7))
-print('IC for column 7: %0.2f' % align_info.ic_vector[7])
-print('test print_info_content')
-AlignInfo.print_info_content(align_info)
-print("testing reading and writing fasta format...")
-
-to_parse = os.path.join(os.curdir, 'Quality', 'example.fasta')
-
-alignment = AlignIO.read(to_parse, "fasta",
-                         alphabet=Alphabet.Gapped(IUPAC.ambiguous_dna))
-
-# test the base alignment stuff
-print('all_seqs...')
-for seq_record in alignment:
-    print('description: %s' % seq_record.description)
-    print('seq: %r' % seq_record.seq)
-
-print('length: %i' % alignment.get_alignment_length())
-align_info = AlignInfo.SummaryInfo(alignment)
-consensus = align_info.dumb_consensus(ambiguous="N", threshold=0.6)
-assert isinstance(consensus, Seq)
-print('consensus: %r' % consensus)
-
-print(alignment)
-
 
-print("Test format conversion...")
-
-# parse the alignment file and get an aligment object
-alignment = AlignIO.read(os.path.join(os.curdir, 'Clustalw', 'opuntia.aln'),
-                         'clustal')
-
-print("As FASTA:")
-print(alignment.format("fasta"))
-print("As Clustal:")
-print(alignment.format("clustal"))
-
-"""
-# test to find a position in an original sequence given a
-# column position in an alignment
-print("Testing finding column positions...")
-alignment_info = ["GATC--CGATC--G",
-                  "GA--CCCG-TC--G",
-                  "GAT--CC--TC--G"]
-
-gapped_unambiguous = Alphabet.Gapped(IUPAC.unambiguous_dna)
-
-alignment = Alignment(gapped_unambiguous)
-for seq in alignment_info:
-    alignment.add_sequence("Blah", seq)
-
-test_seq_1 = Seq("GATCCGATCG")
-orig_pos = alignment.original_sequence_pos(3, test_seq_1, 0)
-assert orig_pos == 3, "Got unexpected position: %s" % orig_pos
-orig_pos = alignment.original_sequence_pos(7, test_seq_1, 0)
-assert orig_pos == 5, "Got unexpected position: %s" % orig_pos
-orig_pos = alignment.original_sequence_pos(0, test_seq_1, 0)
-assert orig_pos == 0, "Got unexpected position: %s" % orig_pos
-orig_pos = alignment.original_sequence_pos(13, test_seq_1, 0)
-assert orig_pos == 9, "Got unexpected position: %s" % orig_pos
-
-try:
-    orig_pos = alignment.original_sequence_pos(5, test_seq_1, 0)
-    raise AssertionError("Did not fail with a junk position")
-except AssertionError:
-    pass
-"""
+# flake8 doesn't like blank lines that contain whitespace.
+# However, such lines may appear in output generated by Clustalw, as in
+# the test test_read_clustal2 below. Therefore we switch off flake8:
+# flake8: noqa
```

Comment:
Rather than a blanket ignore of all validation, please use ``#noqa: W291`` at the end of the handful of multi-line strings doing this.

e.g. https://github.com/biopython/biopython/blob/biopython-173/Tests/test_AlignIO_ClustalIO.py#L38